config/pepperConfig.js.ctmpl

```diff
@@ -197,17 +197,63 @@ var DDP_ENV = {
 {{end}}
 
 {{if eq $study_guid "rgp"}}
+    {{if eq $environment "local"}}
+        DDP_ENV['auth0ClientId'] = "Bal2RZDbCcWHV5W2QgenIWRRN31GGO52";
+        DDP_ENV['auth0Domain'] = "rgp-dev.auth0.com";
+        DDP_ENV['auth0Audience'] = "rgp-dev.auth0.com"
+        DDP_ENV['basePepperUrl'] = "https://pepper-dev.datadonationplatform.org";
+        DDP_ENV['projectGAToken'] = ""fixme-not-set";";
+    {{end}}
     {{if eq $environment "dev"}}
+        DDP_ENV['auth0ClientId'] = "cbI1pq5iRnb4jz4K2atiOmuwGuObaPn1";
+        DDP_ENV['auth0Domain'] = "prion-dev.auth0.com";
+        DDP_ENV['auth0Audience'] = "prion-dev.auth0.com"
+        DDP_ENV['basePepperUrl'] = "https://pepper-dev.datadonationplatform.org";
+        DDP_ENV['projectGAToken'] = ""fixme-not-set";";
+    {{end}}
+    {{if eq $environment "staging"}}
         DDP_ENV['auth0ClientId'] = "fixme-not-set";
+        DDP_ENV['auth0Domain'] = "fixme-not-set";
+        DDP_ENV['auth0Audience'] = "fixme-not-set";
+        DDP_ENV['basePepperUrl'] = "fixme-not-set";
         DDP_ENV['projectGAToken'] = "fixme-not-set";
-    {{else if eq $environment "test"}}
+    {{end}}
+    {{if eq $environment "prod"}}
         DDP_ENV['auth0ClientId'] = "fixme-not-set";
+        DDP_ENV['auth0Domain'] = "fixme-not-set";
+        DDP_ENV['auth0Audience'] = "fixme-not-set"
+        DDP_ENV['basePepperUrl'] = "fixme-not-set";
         DDP_ENV['projectGAToken'] = "fixme-not-set";
-    {{else if eq $environment "staging"}}
+    {{end}}
+{{end}}
+
+{{if eq $study_guid "rgp"}}
+    {{if eq $environment "local"}}
+        DDP_ENV['auth0ClientId'] = "Bal2RZDbCcWHV5W2QgenIWRRN31GGO52";
+        DDP_ENV['auth0Domain'] = "rgp-dev.auth0.com";
+        DDP_ENV['auth0Audience'] = "rgp-dev.auth0.com"
+        DDP_ENV['basePepperUrl'] = "https://pepper-dev.datadonationplatform.org";
+        DDP_ENV['projectGAToken'] = ""fixme-not-set";";
+    {{end}}
+    {{if eq $environment "dev"}}
+        DDP_ENV['auth0ClientId'] = "cbI1pq5iRnb4jz4K2atiOmuwGuObaPn1";
+        DDP_ENV['auth0Domain'] = "prion-dev.auth0.com";
+        DDP_ENV['auth0Audience'] = "prion-dev.auth0.com"
+        DDP_ENV['basePepperUrl'] = "https://pepper-dev.datadonationplatform.org";
+        DDP_ENV['projectGAToken'] = "fixme-not-set";
+    {{end}}
+    {{if eq $environment "staging"}}
         DDP_ENV['auth0ClientId'] = "fixme-not-set";
+        DDP_ENV['auth0Domain'] = "fixme-not-set";
+        DDP_ENV['auth0Audience'] = "fixme-not-set";
+        DDP_ENV['basePepperUrl'] = "fixme-not-set";
         DDP_ENV['projectGAToken'] = "fixme-not-set";
-    {{else if eq $environment "prod"}}
+    {{end}}
+    {{if eq $environment "prod"}}
         DDP_ENV['auth0ClientId'] = "fixme-not-set";
+        DDP_ENV['auth0Domain'] = "fixme-not-set";
+        DDP_ENV['auth0Audience'] = "fixme-not-set"
+        DDP_ENV['basePepperUrl'] = "fixme-not-set";
         DDP_ENV['projectGAToken'] = "fixme-not-set";
     {{end}}
 {{end}}
```

Comment:
It looks like config for RGP appears here twice, could you please double-check.
And usually, studies have 4 following environments: dev, test, staging, prod.

Reply:
You're right--I could have sworn I fixed this but maybe I accidentally reverted the change.

I can do that.  Gen2 RGP just has dev, staging, and prod, but adding one more environment is easy enough.